<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>R. Reucher</author>
 <class>DetailSetup</class>
 <widget class="QDialog" name="DetailSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detail setup</string>
  </property>
  <layout class="QGridLayout" name="gridLayoutBase">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="5">
      <widget class="QSplitter" name="hSplitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="QGroupBox" name="groupBoxAvailableDetails">
        <property name="title">
         <string>Available details</string>
        </property>
        <layout class="QGridLayout" name="gridLayoutAvailableDetails">
         <item row="0" column="0">
          <widget class="QListWidget" name="listWidgetAvailableDetails">
           <property name="toolTip">
            <string>List of available details</string>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButtonConfigureDetail">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Configure current detail</string>
             </property>
             <property name="text">
              <string>Configure...</string>
             </property>
             <property name="icon">
              <iconset resource="qmc2.qrc">
               <normaloff>:/data/img/configure.png</normaloff>:/data/img/configure.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonActivateDetails">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Activate selected details</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="qmc2.qrc">
               <normaloff>:/data/img/right_action.png</normaloff>:/data/img/right_action.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBoxActiveDetails">
        <property name="title">
         <string>Active details</string>
        </property>
        <layout class="QGridLayout" name="gridLayoutActiveDetails">
         <item row="0" column="0">
          <widget class="QListWidget" name="listWidgetActiveDetails">
           <property name="toolTip">
            <string>List of active details and their order</string>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pushButtonDeactivateDetails">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Deactivate selected details</string>
             </property>
             <property name="icon">
              <iconset resource="qmc2.qrc">
               <normaloff>:/data/img/left_action.png</normaloff>:/data/img/left_action.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonDetailsUp">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Move selected detail up</string>
             </property>
             <property name="icon">
              <iconset resource="qmc2.qrc">
               <normaloff>:/data/img/up_action.png</normaloff>:/data/img/up_action.png</iconset>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonDetailsDown">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Move selected detail down</string>
             </property>
             <property name="icon">
              <iconset resource="qmc2.qrc">
               <normaloff>:/data/img/down_action.png</normaloff>:/data/img/down_action.png</iconset>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButtonOk">
       <property name="toolTip">
        <string>Apply detail setup and close dialog</string>
       </property>
       <property name="text">
        <string>&amp;Ok</string>
       </property>
       <property name="icon">
        <iconset resource="qmc2.qrc">
         <normaloff>:/data/img/yes.png</normaloff>:/data/img/yes.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButtonApply">
       <property name="toolTip">
        <string>Apply detail setup</string>
       </property>
       <property name="text">
        <string>&amp;Apply</string>
       </property>
       <property name="icon">
        <iconset resource="qmc2.qrc">
         <normaloff>:/data/img/filesave.png</normaloff>:/data/img/filesave.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="toolTip">
        <string>Cancel detail setup and close dialog</string>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="qmc2.qrc">
         <normaloff>:/data/img/no.png</normaloff>:/data/img/no.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6"/>
 <tabstops>
  <tabstop>listWidgetAvailableDetails</tabstop>
  <tabstop>listWidgetActiveDetails</tabstop>
  <tabstop>pushButtonConfigureDetail</tabstop>
  <tabstop>pushButtonActivateDetails</tabstop>
  <tabstop>pushButtonDeactivateDetails</tabstop>
  <tabstop>pushButtonDetailsUp</tabstop>
  <tabstop>pushButtonDetailsDown</tabstop>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonApply</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="qmc2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>DetailSetup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>DetailSetup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
